pkg/utils/k8sutil.go

```diff
@@ -171,14 +123,14 @@ func GetRestClient() (*rest.RESTClient, error) {
 	return restClient, nil
 }
 
-// GetTPRClient returns tpr client to the request from inside of cluster
-func GetTPRClient() (*rest.RESTClient, error) {
+// GetCRDClient returns tpr client to the request from inside of cluster
+func GetCRDClient() (*rest.RESTClient, error) {
 	tprconfig, err := rest.InClusterConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	configureClient(tprconfig)
+	configureClient("k8s.io", "v1", "/apis", tprconfig)
```

Comment:
Don't modify the returned value from `InClusterConfig()` - it might be cached/reused elsewhere.  (See client-go code for examples of shallow copying first)